20240531_reconstructed_cmblens/README.md

```diff
@@ -0,0 +1,37 @@
+# Reconstucted CMB lensing fields
+
+Louis Legrand (louis.legrand@outlook.com)
+
+The lensing fields are reconstructed from the simulated chwide component separated maps, stored in `/global/cfs/cdirs/cmbs4xlb/v1/component_separated/chwide`
```

Comment:
I might have missed this, do you have some docs about these component  separated maps? can you add it here?

Reply:
These maps were provided by Shamik, but I think there is no docs on this repo yet. Maybe the best would be to ask him to upload it here?

Reply:
@1cosmologist could you create another folder on this repo with docs about those maps?

Reply:
@1cosmologist could you create another folder on this repo with docs about those maps?

Reply:
@zonca Created PR https://github.com/litebird/litebirdXS4/pull/18 to add the documentation for the NILC maps.